src/Bitcoin/SigningInput.h

```diff
@@ -0,0 +1,68 @@
+// Copyright © 2017-2021 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "Amount.h"
+#include "Transaction.h"
+#include "UTXO.h"
+#include <TrustWalletCore/TWBitcoinSigHashType.h>
+#include "../proto/Bitcoin.pb.h"
+
+#include <string>
+#include <vector>
+#include <map>
+#include <optional>
+
+namespace TW::Bitcoin {
+
+/// Input for signing, info of an unsigned transaction
+class SigningInput {
```

Comment:
I guess we can break signing input into smaller classes? group by functionalities etc, no limit like the single proto model